### DISCUSSION TITLE
Get only the URL of the video?

### DISCUSSION THREAD

**Discussion by obviyus** [elmoiv/redvid#29] [Q&A] [answered]
Is there a way to grab the URL of the video in a link? I tried to use `reddit.scrape()` but that gives me a:
```bash
ERROR 'NoneType' object has no attribute 'text'
```

Alternatively, I tried using `reddit.check()` and grabbing the URL from `reddit.video` but that (as expected) has no sound. From the Reddit API I only see a `scrubber_media_url` and `hls_url`. Is there no way to grab a complete file (audio + video) without downloading it?

Thanks for the module, love using it.

**Comment by elmoiv** [comment, marked as answer]
Unfortunately No.

Reddit video player plays audio and video in 2 separate channels, that's why any Reddit video downloader implements a way to combine audio and video using native way or FFmpeg.

If Reddit updated the api with such direct URL, I think there will be no need for any downloaders.
